administrator/components/com_joomlaupdate/Model/UpdateModel.php

```diff
@@ -94,15 +95,18 @@ public function applyUpdateSite()
 				$updateURL = 'https://update.joomla.org/core/list.xml';
 		}
 
+		$id = ExtensionHelper::getExtensionRecord('files_joomla')->extension_id;
 		$db = $this->getDbo();
 		$query = $db->getQuery(true)
 			->select($db->quoteName('us') . '.*')
-			->from($db->quoteName('#__update_sites_extensions') . ' AS ' . $db->quoteName('map'))
+			->from($db->quoteName('#__update_sites_extensions', 'map'))
 			->join(
-				'INNER', $db->quoteName('#__update_sites') . ' AS ' . $db->quoteName('us')
-				. ' ON (us.update_site_id = map.update_site_id)'
+				'INNER',
+				$db->quoteName('#__update_sites', 'us'),
+				$db->quoteName('us.update_site_id') . ' = ' . $db->quoteName('map.update_site_id'),
```

Comment:
```suggestion
				$db->quoteName('us.update_site_id') . ' = ' . $db->quoteName('map.update_site_id')
```
Trailing commas in function calls need PHP 7.3+.